libsolidity/formal/CVC4Interface.cpp

```diff
@@ -132,8 +132,17 @@ CVC4::Expr CVC4Interface::toCVC4Expr(Expression const& _expr)
 		else if (n == "false")
 			return m_context.mkConst(false);
 		else
-			// We assume it is an integer...
+		{
```

Comment:
You could get rid of these parenthesis, if you move the ``return`` into the ``try`` block...